hadoop-ozone/dist/src/main/smoketest/security/ozone-secure-tenant.robot

```diff
@@ -66,13 +66,18 @@ Verify Bucket 1 Owner
     ${result} =         Execute          ozone sh bucket info /tenantone/bucket-test1 | jq -r '.owner'
                         Should Be Equal  ${result}       testuser
 
-Put, get and delete a key in the tenant bucket
+Put a key in the tenant bucket
                         Execute                 echo "Randomtext" > /tmp/testfile
                         Execute and checkrc     aws s3api --endpoint-url ${S3G_ENDPOINT_URL} put-object --bucket bucket-test1 --key mykey --body /tmp/testfile  0
+
+Verify Object Owner
+    ${result} =         Execute          ozone sh key info /tenantone/bucket-test1/mykey | jq -r '.owner'
+                        Should Be Equal  ${result}       testuser
```

Comment:
I am fine with this addition. Just curious, IIUC because Ozone was faking key owner before, this check would still pass without key owner right?

Reply:
Prior to this PR, the Ozone key info command would not display the `owner`, and if it did, it would display `hadoop`
Because docker's startup user is hadoop, so this would have returned hadoop

```bash
[root@centos ~/ozone]$ docker exec ozonesecure-scm-1 whoami
hadoop
[root@centos ~/ozone]$
```